server/chat-plugins/random-battles.ts

```diff
@@ -614,21 +614,18 @@ export const commands: Chat.ChatCommands = {
 
 		const args = target.split(',');
 		if (!args[0]) return this.parse(`/help randombattles`);
-		let dex = Dex;
-		let isLetsGo = false;
-		if (args[1] && toID(args[1]) in Dex.dexes) {
-			dex = Dex.dexes[toID(args[1])];
-			if (toID(args[1]) === 'letsgo') isLetsGo = true;
-		} else if (room?.battle) {
-			const format = Dex.formats.get(room.battle.format);
-			dex = Dex.mod(format.mod);
-			if (format.mod === 'letsgo') isLetsGo = true;
+
+		let {dex} = this.splitFormat(target, true);
+		if (!FS(`${dex.dataDir}/random-teams.js`).readIfExistsSync()) {
```

Comment:
No sync filesystem access in the main process, please!

Also are you reading the entire file just to check if it exists?

Reply:
I'm actually not sure what this is for in general...

Reply:
> No sync filesystem access in the main process, please!

I think some people reading this might point out there's a decent amount of it right now, so let me clarify that it's fine for initialization because it simplifies a lot of code. But I don't want it outside of initialization.